### ISSUE TITLE
update to 1.20.5/6

### ISSUE THREAD

**Issue by Fantomrat** [RubixDev/EnchantedShulkers#56]
There are any plans to update to 1.20.5/1.20.6?

**Comment by RubixDev** [comment]
Yes, and I hope to get to it soon, but 1.20.5 has changed a lot of stuff that this mod depends on so it might take a while.
